.github/workflows/tests.yml

```
@@ -80,11 +80,14 @@ jobs:
             registry: ghcr.io
             username: ${{ github.actor }}
             password: ${{ secrets.GITHUB_TOKEN }}
+        - name: Sanitize Docker tag
+          id: vars
+          run: echo "TAG=$(echo ${{ github.ref_name }} | tr '/' '-')" >> $GITHUB_ENV
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2086:info:1:59: Double quote to prevent globbing and word splitting

(shellcheck)